.travis.yml

```diff
@@ -1,13 +1,32 @@
 language: cpp
 compiler: gcc
+dist: trusty
 
-dist: xenial
 addons:
   apt:
+    sources:
+    - sourceline: 'ppa:beineri/opt-qt593-trusty'
     packages:
-    - libboost-dev
-    - qt5-default
+    - qt59base
+    - libboost1.55-dev
 
 script:
+  - source /opt/qt*/bin/qt*-env.sh
   - qmake
   - make -j$(nproc)
+  - make INSTALL_ROOT=appdir -j$(nproc) install ; find appdir/
+  - wget -c -nv "https://github.com/probonopd/linuxdeployqt/releases/download/continuous/linuxdeployqt-continuous-x86_64.AppImage"
+  - chmod a+x linuxdeployqt-continuous-x86_64.AppImage
+  - unset QTDIR; unset QT_PLUGIN_PATH ; unset LD_LIBRARY_PATH
+  - export VERSION=$(git rev-parse --short HEAD) # linuxdeployqt uses this for naming the file
+  - ./linuxdeployqt-continuous-x86_64.AppImage appdir/usr/share/applications/*.desktop -appimage
```

Comment:
If we are to use the SVG icon as the window icon, don't we need to ensure that the SVG icon engine is part of the AppImage?

Reply:
Great catchg! Let me give it a try